<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1" rowspan="2" colspan="2">
     <widget class="QLabel" name="searchingInfoLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="findButton">
      <property name="text">
       <string>Найти элемент</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="searchingSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Элементов:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="amountOfElementsSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sortButton">
      <property name="text">
       <string>Сортировать</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="3" column="1" rowspan="2" colspan="2">
     <widget class="QLabel" name="sortingInfoLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QPushButton" name="fillButton">
      <property name="text">
       <string>Заполнить случайно</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="sortingComboBox">
      <item>
       <property name="text">
        <string>слиянием (Merge Sort)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>быстро (Quick Sort)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>кучей (Heap Sort)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>интерполяционно (Interpolation sort)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Модуль:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="modSpinBox">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
